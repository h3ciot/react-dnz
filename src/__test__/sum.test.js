function sum(a, b) {
  return a + b;
}

test('add 1+2 to equal 3', () => {
  expect(sum(1, 2)).toMatchSnapshot();
  expect(sum(1, 2)).toBe(3);
});
