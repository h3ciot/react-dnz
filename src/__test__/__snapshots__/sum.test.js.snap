// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add 1+2 to equal 3 1`] = `3`;
